src/tests/multihost/alltests/test_proxy_rfc2307bis.py

```diff
@@ -414,16 +381,9 @@ def test_simple_deny_groups_user_cs2_grp1(self, multihost, backupsssdconf):
                 client.login(login_timeout=10, sync_multiplier=1,
                              auto_prompt_reset=False)
             time.sleep(3)
-            execute_cmd(multihost, f'cat /var/log/secure | grep "Access denied for user {user}"')
+            check_logs(multihost, user)
```

Comment:
Here we are asserting so better to keep at end of test case.
same for line 346.

Reply:
This is not possible , on the next line we can see logs are being truncated

Reply:
Its possible, return the method and assert in your statements.

Reply:
def check_logs(multihost, user):
    log_str = multihost.client[0].get_file_contents("/var/log/secure").decode('utf-8')
    assert f"Authentication failure for illegal user {user}" in log_str\
           or f"Access denied for user {user}" in log_str


How is it possible ?

Reply:
def check_logs(multihost, user):
log_str = multihost.client[0].get_file_contents("/var/log/secure").decode('utf-8')
return log_str

in test case call assert statement.

Reply:
but within  for loop how you are going to do that ?